exonum/src/runtime/mod.rs

```diff
@@ -187,6 +187,10 @@ pub trait Runtime: Send + Debug + 'static {
 
     /// Configure a service instance with the given parameters.
     ///
+    /// The configuration parameters passed to the method are not saved anywhere,
+    /// so the service instance should save them by itself if it is important for
```

Comment:
It seems important that this method is only called once (or once per configuration change, when that is implemented) to understand the need to make some changes to persistent storage in the implementation of this method (not necessarily save them as is, but update the state based on them), therefore, I'd consider moving this paragraph _after_ the subsequent.

Reply:
On the other hand — these are internal docs, so any order will do here, but in `Service` docs it must be very clear.